client/src/assets/icons/moon-fill.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" viewBox="0 0 16 16">
```

Comment:
My guess is that these icons are public domain or similar.

Shouldn't we document in the commit or PR where these icons come from?